packages/shared/src/utils/constants/contract.tsx

```diff
@@ -137,18 +137,18 @@ export const getContractTypesConfig: TGetContractTypesConfig = symbol => ({
         config: { hide_duration: true },
     }, // hide Duration for Multiplier contracts for now
     turboslong: {
-        title: localize('Long'),
+        title: localize('Turbos'),
         trade_types: ['TURBOSLONG'],
         basis: ['stake'],
         barrier_count: 1,
-        components: ['barrier_selector', 'take_profit'],
+        components: ['trade_type_tab', 'barrier_selector', 'take_profit'],
```

Comment:
```suggestion
        components: ['trade_type_tabs', 'barrier_selector', 'take_profit'],
```
@hamza-deriv let's rename to point out there are more than 2 of them😊

Reply:
Done